akka-http-core/src/main/java/akka/http/javadsl/model/headers/TE.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model.headers;
+
+/**
+ *  Model for the `Transfer-Encoding` header.
+ *  Specification: http://tools.ietf.org/html/draft-ietf-httpbis-p1-messaging-26#section-3.3.1
+ */
+public abstract class TE extends akka.http.scaladsl.model.HttpHeader {
+    public abstract Iterable<akka.http.javadsl.model.TransferEncoding> getAcceptedEncodings();
```

Comment:
The implementation seem to be missing?

Reply:
doh, I'm clearly trying to do too much stuff in parallel here..